packages/components/src/Overlay/Popover.tsx

```diff
@@ -20,7 +20,10 @@ export interface PopoverProps
 // Component
 // ---------------
 const _Popover = forwardRef<HTMLDivElement, PopoverProps>(
-  ({ keyboardDismissDisabled, placement, open, children, ...rest }, ref) => {
+  (
+    { keyboardDismissDisabled, placement, open, className, children, ...rest },
```

Comment:
As far as I see you don't use the **className** prop somewhere or? So you can undo it and remove it from props again in L.14?